test/server.js

```diff
@@ -14,9 +14,17 @@ const register = client.register;
 
 const exec = require('child_process').exec;
 const env = require('./environment');
+const logger = require('./helpers/logger');
 
 const testOutputPath = `${__dirname}/test_reports/junit/e2e-results.xml`;
 
+// 80/443 standard http/https
+// do we need to allow this to be configured?
+// need to ensure the var names are not confusing,
+// this is not the port for the console, this is for
+// the /metrics endpoint
+const METRICS_SERVER_PORT = 3000;
```

Comment:
Regarding the issue we experienced with the processing the envVar, I think if we want to have the METRICS_SERVER_PORT variable configurable it should be set in `environment.js` where we do other configurations. 
Thoughts ?

Reply:
I think we don't need it configurable, so I removed the check for `process.env.PORT`.  Seems overkill doesn't it?

Reply:
Make sense :+1: